### ISSUE TITLE
Screenreaders and Notifications using Firefox

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/bigbluebutton#12572]
I am a blind user and first of all I am thankful for all the work that has already been done to make BBB as accessible as it is. I have found an issue while using BBB with Firefox. 

Current behaviour:
When all notifications are turned on and someone joins the room. I can only hear the notification sound. The Screenreader doesn't announce the pop-up. 

Expected behaviour:
When a Notification takes place I'd love the Screenreader announce what happens. 

Maybe some helpful stuff: 
In Chrome everything works just fine. In Firefox there is one time where the notification comes throuch. That is when someone raises their hand. Maybe it is possible to change the other notifications to the same system as in the "raising hand"-Notification?

**Comment by antobinary** [comment]
Hi @WestphalDenn 
Thank you for bringing this up!

Could you please confirm what version of BigBlueButton you are using? The recommended version is 2.3 (2.3.3 released about ten days ago)

**Comment by WestphalDenn** [comment]
I am using 2.3.3. Just confirmed as well that the notifications for raised hands are coming through just fine in Firefox and get announced. If whatever is different there get transfered to the other messages e.g. when someone enters the conference, that should fix the issue. At least I'd think so. 

hth
